.travis.yml

```diff
@@ -5,9 +5,8 @@ dist: trusty
 sudo: required
 
 go:
-  - 1.8
-  - 1.9
```

Comment:
Buffalo's next version will support Go 1.9, so we can't drop it now: https://github.com/gobuffalo/buffalo/issues/1106

Reply:
Despite go 1.9 being past its EOL now?

Reply:
Yes. You can open an issue in the buffalo repo if you think we should reconsider it, but from the time being Buffalo will target 1.9+ on the next version.

Reply:
Btw., I've updated the commit yesterday to re-include 1.9